src/test/java/core/basesyntax/ShopServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.shopservice;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.OperationStrategy;
+import core.basesyntax.operation.OperationStrategyImpl;
+import core.basesyntax.operation.handler.BalanceOperation;
+import core.basesyntax.operation.handler.PurchaseOperation;
+import core.basesyntax.operation.handler.ReturnOperation;
+import core.basesyntax.operation.handler.SupplyOperation;
+import core.basesyntax.storage.Storage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ShopServiceImplTest {
+    private final String testFruit = "apple";
+    private final int testQuantity = 13;

```

Comment:
Make those static constants:
```suggestion
    private static final String TEST_FRUIT = "apple";
    private static final int TEST_QUANTITY = 13;

```